### PR TITLE
Task 309-311: RACE

### PR DESCRIPTION
**Pull request by Mirzyaaliii**
task309_RACE_answer_generation  -->  Given an article, a question and four options; provide the correct answer for the question based on the article.
task310_RACE_classification  -->  Given an article, a question and four options; provide the incorrect answer for the question based on the article.
task311_RACE_question_generation  -->  Generate a question based on the given article and an answer.

Please review.

### CODE REVIEW

**Comment by swarooprm** [comment]
@nrjvarshney can you review these tasks?

**Comment by swarooprm** [comment]
> @nrjvarshney can you review these tasks?

@nrjvarshney, pinging in this thread to check if you have seen my earlier comment?

File: tasks/README.md [hunk 1]
```diff
@@ -228,4 +228,7 @@ Name | Summary | Category
 `task273_europarl_classification.json`| Given bulgarian sentence and corresponding english translation, verify that the translation is right or wrong | Classification
 `task274_overruling_classification`	| Given a sentence, classify it into overruling or non-overruling| Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
-`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
\ No newline at end of file
+`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task309_RACE_answer_generation` | Given an article, a question and four options; provide correct answer for the question based on the article. | Answer Generation
```
Comment (danyaljj) [inline tasks/README.md]: we gotta lowercase these names and merge with the main branch.

Comment (Mirzyaaliii) [reply tasks/README.md]: Done.

**Comment by Mirzyaaliii** [comment]
> > @nrjvarshney can you review these tasks?
> 
> @nrjvarshney, pinging in this thread to check if you have seen my earlier comment?

@danyaljj or @swarooprm please assign this PR to someone else.

**Comment by swarooprm** [comment]
@Maitreyapatel, wondering if you can help review this PR?

**Review by swarooprm — COMMENTED** [review]
@Mirzyaaliii, looks good overall, some comments:

- Task309: you're given a article, a question or fill-in-the-blank--> you are given an article, a question which often contains a blank. In the definition, mention to generate only the option names and not the exact answer. Ask not to generate anything else apart from A, B, C, D. Also, optionally specify that, only one of the 4 options are correct.
- Task 310- same changes as 309.
- Task 311- do you have multiple questions per a single input? If yes, append them all and make it a list in instance outputs (output of positive example will still be a single question). Multiple questions improve the reliability of the evaluation. I also have a problem when you say "task is to generate the question (which can be in form of fill-in-the-blank also) for this answer based on the given article." Don't give options because if someone generates fill-in-the-blank question and we don't have that in our gold annotation, the evaluation will be incorrect. I would suggest splitting this in to 2 tasks (i) ask to generate a question (by default question does not mean fill in the blank) (ii)ask to generate fill in the blank question.
Some minor typos: you're given a article and an answer. You task is to generate the question (which can be in form of fill-in-the-blank also) for this answer based on the given article--->you are given an article and an answer. Your task is to generate a question for the provided answer based on the given article.

If you want, you can create 2 more tasks (i) generate correct answer (without options) and (ii) generate incorrect answer (without options) similar to task003 and task004.

**Comment by Mirzyaaliii** [comment]
> @Mirzyaaliii, looks good overall, some comments:
> 
> * Task309: you're given a article, a question or fill-in-the-blank--> you are given an article, a question which often contains a blank. In the definition, mention to generate only the option names and not the exact answer. Ask not to generate anything else apart from A, B, C, D. Also, optionally specify that, only one of the 4 options are correct.
> * Task 310- same changes as 309 in definition.
> * Task 311- do you have multiple questions per a single input? If yes, append them all and make it a list in instance outputs (output of positive example will still be a single question). Multiple questions improve the reliability of the evaluation. I also have a problem when you say "task is to generate the question (which can be in form of fill-in-the-blank also) for this answer based on the given article." Don't give options because if someone generates fill-in-the-blank question and we don't have that in our gold annotation, the evaluation will be incorrect. I would suggest splitting this in to 2 tasks (i) ask to generate a question (by default question does not mean fill in the blank) (ii)ask to generate fill in the blank question.
>   Some minor typos: you're given a article and an answer. You task is to generate the question (which can be in form of fill-in-the-blank also) for this answer based on the given article--->you are given an article and an answer. Your task is to generate a question for the provided answer based on the given article.
> 
> If you want, you can create 2 more tasks (i) generate correct answer (without options) and (ii) generate incorrect answer (without options) similar to task003 and task004.
> 
> Also, in all these tasks. I assume labels are balanced.


Thank you very much for your comments @swarooprm.
- In Task 309 and 310, changes are done according to suggestions.
- From task 311, as suggested I separate the task for the question and fill-in-the-blank question. Please check #309 for another task.

Let me know if any other changes are required.

**Review by swarooprm — APPROVED** [review]
LGTM! Merging.

**Pull request merged**